README.md

```diff
@@ -48,37 +18,40 @@ So that I am reassured that my order will be delivered on time
-  * The text sending functionality should be implemented using Twilio API. You'll need to register for it. It’s free.
-  * Use the twilio-ruby gem to access the API
-  * Use the Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-  * Note that you can only send texts in the same country as you have your account. I.e. if you have a UK account you can only send to UK numbers.
+## Domain Modelling
 
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
+The Takeaway app is split up into various classes that understand the behaviour of various methods. The classes encapsulate these behaviours. 
 
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
+The Takeaway app is based on my favourite takeaway option - Hawaiian Poké Bowls!
 
-> :warning: **WARNING:** think twice before you push your **mobile number** or **Twilio API Key** to a public space like GitHub :eyes:
->
-> :key: Now is a great time to think about security and how you can keep your private information secret. You might want to explore environment variables.
+The takeaway class understands the behaviour of the view_menu, add_item and view_order methods.
 
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
+- Class ----> takeaway
+- Method (behaviour) ----> view_menu, add_item and view_order 
 
+The order class understands the behaviours of the add_to_order, order_summary and basket_total methods.
 
-In code review we'll be hoping to see:
+- Class ----> order
+- Method (behaviour) ----> add_to_order, order_summary and basket_total
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
+The menu class understands the behaviour of the menu method.
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
+- Class ----> menu
+- Method (behaviour) ----> menu
 
-Notes on Test Coverage
-------------------
+## How to run
 
-You can see your [test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) when you run your tests.
+```
+irb
```

Comment:
using an irb example in the readme makes it really easy to understand how your program works, as does the domain modelling